src/Likvido.Configurations/ServiceConfiguration.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+
+namespace Likvido.Configurations
+{
+    public static class ServiceConfiguration
```

Comment:
It would be nice if we can get rid of this one - and just have the other configuration builder